### PR TITLE
chore: CRW-4582 add more exceptions for known issues

### PR DESCRIPTION
**Pull request by nickboldt**
chore: [CRW-4582](https://issues.redhat.com//browse/CRW-4582) add more exceptions for known issues

Signed-off-by: Nick Boldt <nboldt@redhat.com>

### CODE REVIEW

File: dist/releases/4.10/config.toml [hunk 1]
```diff
@@ -52,3 +52,13 @@ filter_files = [
...
 ]
+
+[payload.devspaces-udi-rhel8-container]
+filter_files = [
+  ### include golang compiler intentionally; exception needed until https://issues.redhat.com/browse/GT-130 is resolved
+  "/usr/lib/golang/pkg/tool/linux_amd64/compile",
+  "/usr/lib/golang/pkg/tool/linux_ppc64le/compile",
+  "/usr/lib/golang/pkg/tool/linux_s390x/compile",
+  ### helm is not FIPS compliant
+  "/usr/bin/helm"
```
Comment (rphillips) [inline dist/releases/4.10/config.toml:63]: We should be able to rebuilt helm with the correct setting?

Comment (nickboldt) [reply dist/releases/4.10/config.toml:63]: sure but who is "we" and when will that happen?

Comment (rphillips) [reply dist/releases/4.10/config.toml:63]: Can we remove it from the image?

Comment (nickboldt) [reply dist/releases/4.10/config.toml:63]: technically, yes. But afaik customers are using it.

**Pull request merged**